lib/canvas_exporting.rb

```diff
@@ -0,0 +1,7 @@
+require 'canvas_exporting/version'
+require 'canvas_exporting/exporter'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.